ui/scripts/modal.js

```diff
@@ -1,18 +1,29 @@
 export default class Modal {
     // @todo: will need to add trailer url later
-    constructor(title) {
+    constructor(title, trailer_url) {
         this.title = title;
-        // this.url = url;
+        this.trailer_url = trailer_url;
+    }
+
+    getId(url) {
+        const regExp = /^.*(youtu.be\/|v\/|u\/\w\/|embed\/|watch\?v=|&v=)([^#&?]*).*/;
+        const match = url.match(regExp);
+    
+        return (match && match[2].length === 11)
+          ? match[2]
+          : null;
     }
 
     fillInModal() {
-        console.log("hej!")
         const title = document.querySelector(".modal-title");
         title.textContent = this.title;
 
-        // @todo: will have a dynamic link to the trailer later
         const body = document.querySelector(".modal-body");
-        const trailer = this.createTrailer("https://www.youtube.com/embed/LtNYaH61dXY?si=j-aoQsvgBPQFIfBg");
+
+        console.log(`Trailer URL: ${this.trailer_url}`);
```

Comment:
I dunno but - do we need all the console.logs in the code? (Legit question - I don't know what's considered "normal".)

Reply:
We don't, I just missed them during the clean up. Thanks!